docs/statistics.md

```diff
@@ -1,269 +1,425 @@
+TODO: 
 
-## pktSndLossTotal
+- This is wrong -> ticket, correct this and make separate statistic. Decrypted statistic should include wrongly encrypted packets. Here we should save the comment regarding the version 1.4.0-1.4.1 and include the comment that since version 1.5.0 we do not take this into account.
+- Calculate loss for FEC -> ticket (low priority), there is something already
+- We have already pktRcvUndecryptTotal
 
-The total number of data packets considered or reported lost (sender side). Does not correspond to packets detected as lost at receiving side. Applicable for data sender.
+Since SRT v1.4.0, includes packets that failed to be decrypted.
 
-A packet is considered lost in two cases: 
-1. Sender receives a loss report from receiver.
-2. Sender initiates retransmission after not receiving ACK for a certain timeout. Refer to `FASTREXMIT` and `LATEREXMIT` algorithms.
+### pktRetransTotal
 
-## pktRcvLossTotal
+The total number of retransmitted packets. Available for sender.
 
-The total number of data packets detected lost on the receiver's side.
+TODO: 
 
-Includes packets that failed to be decrypted (only as of SRT version 1.4.0).
+- We can calculate retransmitted packets both at the sender side and at the receiver side which is stated in pktSentTotal and pktRecvTotal stats -> we should have this statistics available both for snd and rcv. The same for byteRetransTotal -> rename this to pktSentRetransTotal plus add new statistics pktRecvRetransTotal, be careful this is API change. So, think how we can use two names pktRetransTotal and pktSentRetransTotal, until pktRetransTotal can be deprecated. And here we should put special comment. Create ticket for this.
+- Not exchanged with the receiver: what does it mean?
 
-If a packet was received out of order, the gap (sequence discontinuity) is also
-treated as lost packets, independent of the reorder tolerance value.
+### pktSentACKTotal
 
-Loss detection is based on gaps in Sequence Numbers of SRT DATA packets. Detection
-of a packet loss is triggered by a newly received packet. An offset is calculated
-between sequence numbers of the newly arrived DATA packet and previously received
-DATA packet (the received packet with highest sequence number). Receiving older
-packets does not affect this value. The packets from that gap are considered lost,
-and that number is added to this `pktRcvLossTotal` measurement. In the case where
-the offset is negative, the packet is considered late, meaning that it was either
-already acknowledged or dropped by TSBPD as too late to be delivered. Such late
-packets are ignored.
+The total number of sent ACK packets. Available for sender.
 
-## pktRetransTotal
+TODO: 
 
-The total number of retransmitted packets. Calculated on the sender's side only.
-Not exchanged with the receiver.
+- possible mistake, see pktSentNAKTotal -> receiver side? -> Maxim should check (Available for sender. -> receiver)
```

Comment:
Receiver is the side that sends ACK and LOSSREPORT. That's why this is a stat available at the receiver only.

Conversely, the sender is the side that counts `pktRecvACKTotal` etc.